site/pages/[locale]/token-revokes.js

```diff
@@ -371,102 +398,78 @@ const TokenRevokes = function () {
       <TableBox
         text={t('utilities-text.token-revokes')}
         title={t('list-and-revoke-token-approvals')}
+        className="shadow-lg"
       >
-        {/* {tokenApprovals.length > 0 && (
-          <section className="flex flex-col">
-            <div className="grid grid-cols-4">
-              <span className="m-auto text-gray-600 font-bold">
-                {t('token')}
-              </span>
-              <span className="hidden m-auto text-gray-600 font-bold md:block">
-                {t('spender-address')}
-              </span>
-              <span className="m-auto text-gray-600 font-bold">
-                {t('allowance')} / {t('balance')}
-              </span>
-              <span className="m-auto text-gray-600 font-bold">
-                {t('actions')}
-              </span>
+        <div className="w-100 md:w-150 overflow-auto lg:w-full">
+          <table className="w-150">
+            <thead>
+              <tr className="bg-slate-50 text-slate-600 border-slate-200 rounded-xl border text-left text-sm">
+                <th className="w-10 px-2 py-4 font-medium">{t('token')}</th>
+                <th className="w-14 font-medium">{t('allowance')}</th>
+                <th className="w-14 font-medium">{t('balance')}</th>
+                <th className="w-24 font-medium">{t('spender-address')}</th>
+                <th className="w-10 font-medium">{t('actions')}</th>
+              </tr>
+            </thead>
+            <tbody>
               {tokenApprovals.map(
                 ({ address, allowance, transactionHash, spender }) => (
-                  <React.Fragment key={transactionHash}>
-                    <Token address={address} />
-                    <div className="hidden my-auto md:block">
-                      <Token address={spender} />
-                    </div>
-                    <div className="my-auto">
+                  <tr className="border-b" key={transactionHash}>
+                    <td className="px-2 py-4">
+                      <Token address={address} />
+                    </td>
+                    <td>
                       <Allowance address={address} data={allowance} />
-                      <span> / </span>
+                    </td>
+                    <td>
                       <Balance address={address} />
-                    </div>
-                    <Button
-                      disabled={!active}
-                      onClick={() => handleRevoke(address, spender)}
-                      width="w-28"
-                    >
-                      {t('revoke')}
-                    </Button>
-                  </React.Fragment>
+                    </td>
+                    <td>
+                      <Token address={spender} />
+                    </td>
+                    <td className="px-2 py-4">
+                      <Button
+                        className=""
```

Comment:
It looks like this empty className can be removed
```suggestion
```

Reply:
removed